src/protocol.cpp

```diff
@@ -19,6 +19,8 @@ namespace NetMsgType
 const char* VERSION = "version";
 const char* VERACK = "verack";
 const char* ADDR = "addr";
+const char *ADDRV2="addrv2";
+const char *SENDADDRV2="sendaddrv2";
```

Comment:
nit: pointer/reference on the left side as per clang-format rules:
```cpp
const char* ADDRV2="addrv2";
const char* SENDADDRV2="sendaddrv2";
```

Reply:
changed